docs/configuration/index.rst

```diff
@@ -0,0 +1,75 @@
+Configuration
+=============
+
+This is the list of possible configurations that go in ``conf.py``.
+
+Yosys
++++++
+
+``hdl_diagram_yosys`` tells the program what version or binary of Yosys to use.
+By default it is set to ``YoWASP``. Setting it to ``system``, the program will
+use Yosys available in the current PATH. It can also contain the path to a specific
+Yosys binary.::
+
+    hdl_diagram_yosys = "yowasp" # default
+
+    hdl_diagram_yosys = "system" # use yosys from PATH
+
+    hdl_diagram_yosys = "<path-to-Yosys>" # use specific yosys binary
+
+
+netlistsvg
+++++++++++
+
+netlistsvg can take various skin files for use when creating diagrams. It is
+set to ``default`` by default, using the built-in netlistsvg skin.::
+
+    hdl_diagram_skin = "<path-to-skin>"
+
+
+Output format
++++++++++++++
+
+The output format for the generated diagrams can either be set to ``svg`` or ``png``.::
+
+    hdl_diagram_output_format = "svg"
+
+    hdl_diagram_output_format = "png"
+
+
+GHDL
+++++
+
+GHDL can either be prebuilt into Yosys or loaded at runtime. If it is loaded at
+runtime, then this can either be set to ``module`` if GHDL is in the PATH, or
+a path to the ``ghdl.so`` shared library. ::
+
+    hdl_diagram_ghdl = "module" # default, passes `-m ghdl` to Yosys
+
+    hdl_diagram_ghdl = "prebuilt" # if GHDL is built into Yosys
+
+    hdl_diagram_ghdl = "<path-to-GHDL-shared-library>" # path to specific ghdl.so
+
+
+Common Errors
++++++++++++++
+
+.. code-block::
+
+    ERROR:
+      This version of Yosys cannot load plugins at runtime.
+      Some plugins may have been included at build time.
+      Use option `-H' to see the available built-in and plugin commands.
+
+This error signifies that the current version of Yosys cannot load plugins
+at runtime, and so all plugins must be prebuit. For VHDL, ``hdl_diagram_ghdl``
```

Comment:
Rather than 'prebuilt', I'd use 'built-in' or 'built into Yosys'. I mean, 'ghdl.so' is a prebuilt module, but it's not built-in.